src/python/pants/backend/python/rules/pytest_runner.py

```diff
@@ -287,7 +287,7 @@ async def run_python_test(
     )
 
 
-@rule(desc="Run Pytest in an interactive process")
+@rule(desc="Setup Pytest to run interactively")
```

Comment:
```suggestion
@rule(desc="Set up Pytest to run interactively")
```
"Setup" is a noun, "Set up" is a verb.

Reply:
Not worth rerunning CI for though, so you can also squeeze this into some other change, or skip CI.

Reply:
I'll fix in the upcoming PR to stream `test`, rather than dumping :)